docs/BreakingChanges/099 Binding a WPF selector property (such as 'SelectedItem') to a static property does not work.md

```diff
@@ -0,0 +1,34 @@
+## 99: Binding a WPF selector property (such as 'SelectedItem') to a static property does not work
```

Comment:
{
  "Id": "99",
  "Title": "Binding a WPF selector property (such as 'SelectedItem') to a static property does not work",
  "Details": "In the .NET Framework 4.5, WPF Selector properties (such as 'SelectedItem' on ListBoxes or DataGrids) that are data-bound to static properties do not properly update when their bound property is updated.",
  "Suggestion": "This behavior was reverted in a servicing update for the .NET Framework 4.5. Please update the .NET Framework 4.5, or upgrade to .NET Framework 4.5.1 or later, to fix this issue.",
  "Link": "https://social.msdn.microsoft.com/Forums/vstudio/en-US/56ff3199-5abc-454e-87f7-bafff43e7422/net-45-critical-selecteditem-not-setting-binding-item?forum=wpf",
  "Markdown": null,
  "ApplicableApis": [
    "Not detectable via API analysis"
  ],
  "Related": null,
  "VersionBroken": "4.5",
  "VersionFixed": "4.5",
  "IsBuildTime": false,
  "IsQuirked": false,
  "IsSourceAnalyzerAvailable": false,
  "BugLink": "http://vstfdevdiv:8080/web/wi.aspx?pcguid=22f9acc9-569a-41ff-b6ac-fac1b6370209&id=550386",
  "Notes": "We could look for FrameworkElement.SetBinding calls, but this is such a corner case of that (selected properties and static properties) that it would probably be too noisy. A xaml analyzer is probably our best bet, and that's not even going to be perfect",
  "IsRetargeting": false,
  "ImpactScope": 2
}

**Parsed APIs:**
- [ ] Not detectable via API analysis